config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "zahra120",
```

Comment:
get this from process.ENV.DB_USERNAME because william has a different username in his own local computer.